docs/platform-integration/essentials/geocoding.md

```diff
@@ -13,45 +13,46 @@ The `Geocoding` class provides APIs to geocode a placemark to a positional coord
 
 [!INCLUDE [get-started](includes/get-started.md)]
 
-To access the **Geocoding** functionality the following platform specific setup is required.
+[!INCLUDE [essentials-namespace](includes/essentials-namespace.md)]
+
+To access the **Geocoding** functionality the following platform-specific setup is required.
 
+<!-- markdownlint-disable MD025 -->
 # [Android](#tab/android)
 
-No additional setup required.
+No setup is required.
 
 # [iOS](#tab/ios)
 
-No additional setup required.
+No setup is required.
 
 # [Windows](#tab/windows)
 
-A Bing Maps API key is required to use geocoding functionality. Sign up for a free [Bing Maps](https://www.bingmapsportal.com/) account. Under **My account > My keys** create a new key and fill out information based on your application type (which should be **Public Windows App (UWP, 8.x, and earlier)** for UWP apps).
+A Bing Maps API key is required to use geocoding functionality. Sign up for a free [Bing Maps](https://www.bingmapsportal.com/) account. Under **My account** > **My keys**, create a new key and fill out information based on your application type, which should be **Public Windows App (UWP, 8.x, and earlier)**.
 
-Early on in your application's life before calling any **Geocoding** methods set the API key (which is only available on UWP):
+Early on in your application's life before calling any `Geocoding` methods, set the API key, which is only available on Windows:
 
 ```csharp
 Platform.MapServiceToken = "YOUR-KEY-HERE";
 ```
 
 -----
+<!-- markdownlint-enable MD025 -->
 
-## Using Geocoding
-
-[!INCLUDE [essentials-namespace](includes/essentials-namespace.md)]
+## Use geocoding
```

Comment:
Random comment: Xamarin.Forms had its own geocoding/reverse geocoding API. It'll be interesting to see if it's transferred into MAUI, and whether one the Essentials or MAUI geocoding API is dropped.